python/actions/clone_review_with_latest/hook/clone_review.py

```diff
@@ -54,19 +54,27 @@ def launch(self, session, entities, event):
                 f'AssetVersion where asset_id is {object["asset_version"]["asset_id"]} and is_latest_version is True'
             ).one()
             new_session_object = session.create(
-                "ReviewSessionObject", {"asset_version": latest_version}
+                "ReviewSessionObject", {
+                    "asset_version": latest_version,
+                    "version": f"Version {latest_version['version']}",
+                    "name": latest_version["task"]["name"],
```

Comment:
Close to an edge case but an AssetVersion does not need to be attached to a task, if there is no task I guess this will fail.

Reply:
good point indeed.

Reply:
Confirmed this is the case with files that were dropped directly into review sessions.

```
Traceback (most recent call last):
  File "/Users/runner/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages/ftrack_api/event/hub.py", line 810, in _handle
  File "/Users/runner/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages/ftrack_action_handler/action/base.py", line 176, in _launch
  File "/Users/mattias/Library/Application Support/ftrack-connect-plugins/clone-review-with-latest-0.1.0/hook/clone_review.py", line 60, in launch
    "name": latest_version["task"]["name"],
TypeError: 'NoneType' object is not subscriptable
```